properties_pane/entity_level/entityLevelConfig.json

```diff
@@ -894,7 +900,10 @@ making sure that you maintain a proper JSON format.
 						"propertyTooltip": "comments",
 						"addTimestampButton": false,
 						"propertyType": "details",
-						"template": "textarea"
+						"template": "codeEditor",
```

Comment:
"template": "textarea",
"markdown": true

Reply:
Code editor has "HTML" preview as well, so the issue is resolved